docs/appendix/troubleshooting.md

```diff
@@ -3,227 +3,103 @@ title: "Troubleshooting"
+   - If that fails, try re-imaging your board following the [installation guide](/installation/) appropriate for your board.
+   If using the Raspberry Pi [installation guide](/installation/prepare/rpi-setup/), be sure to carefully enter the configuration details under the **Advanced Options** (gear icon) button on the [Raspberry Pi imager](https://www.raspberrypi.com/software/) before you re-image your board.
+
+### ssh: connect to host hostname port 22: Connection timed out
+
+- **Description:** Your computer is not able to connect to the {{< glossary_tooltip term_id="board" text="board" >}} through `ssh` before reaching a timeout.
+
+- **Solution:** Depending on your local network, it may take a minute or two for your `ssh` connection to successfully reach the board.
+   - If you encounter this error once or twice, try your `ssh` command again and wait until it completes.
+   - If the error persists, try the solutions presented for the `Host is down` error message above.
+
+## Common Viam App Errors
+
+### Failed to connect; retrying
+
+- **Description:** The [Viam app](https://app.viam.com) is unable to communicate with your robot, and will attempt to reconnect every few seconds until it is able to do so.
+When a robot is disconnected, it will continue to run with its locally-cached current configuration, but is not accessible for control or configuration remotely through the Viam app.
+
+- **Solution:** Check the following to ensure your robot is accessible to the Viam app:
+   - Is the {{< glossary_tooltip term_id="board" text="board" >}} component connected to the internet?
+   - Is the `ssh` service configured and running locally on the board?
+     You can check with `sudo systemctl status ssh`. It should be listed as `active (running)`.
+   - Is the `viam-server` service running locally on the board?
+     You can check with `sudo systemctl status viam-server`. It should be listed as `active (running)`.
+      - If it is listed as `stopped` or `failed`, you can try restarting it with `sudo systemctl start viam-server`
+      - If it is not found, be sure you have followed the [installation instructions for your board](https://docs.viam.com/installation/#preparation), and then followed the instructions on the **Setup** tab on the Viam app.
+      - If none of the above succeed in getting `viam-server` up and running, check the logs for any pertinent error messages.
+      Depending on your Linux OS, you might use a command similar to `grep 'viam-server' /var/log/syslog | tail -50` to show the 50 most recent log messages from `viam-server`.
+
+### Error: cannot parse config: JSON: cannot unmarshal string into Go struct
+
+- **Full Error:** `Error: cannot parse config: JSON: cannot unmarshal string into Go struct field Component.components.frame of type float64.`
+
+- **Description:** A [frame](/services/frame-system/) attribute may be malformed, and is preventing the parsing of the component's configuration.
```

Comment:
If we could nail down the technically accurate way to change this, I'd love to know what is parsing it. i.e., "...and is preventing `viam-server` from parsing the component's configuration." But it's probably something else not `viam-server` doing the parsing?

Reply:
This is what I have from engineering on this error:

> What is it/Why it happens: Check if there is a frame attribute in components in fancy config or in raw JSON config. It may be that the robot config generated a broken “frame” information and prepopulated the translation values with empty strings when they are currently programmed to be floats on the backend. 

I opted not to include this, and have a suspicion it no longer applies. Let me ask eng to be sure! Will report back.

Reply:
I don't think we need that level of detail; I was just thinking that what you have here is passive voice-adjacent and maybe it could be just ever so slightly more precise and active.

Reply:
(But also not going to block merging just for this! I should've marked as [opt] :) )